### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by markdfrey**
Hey Ty,

Here are my first draft of changes. I added a /referral index for the referral page, and made some changes to the /articles index page to start cleaning up the look a bit. I didn't change anything on the /articles/show page yet.

-Mark

### CODE REVIEW

File: app/views/articles/index.html.erb [hunk 1]
```diff
@@ -154,13 +154,15 @@
...
 			<a href = <%= article_path(article) %>>
-				<div class = "col-md-4">
+				<!-- <div class = "col-md-4"> -->
+                <div class="col-xs-6 col-md-3">
 					<div>
 						<%= image_tag(article.photo.url(:medium), :class => 'thumbnail') %>
 					</div>
 					<div>
-						<%= article.title %>
+						<h4 class="media-heading"><%= article.title %></h3>
```
Comment (tmclark737) [inline app/views/articles/index.html.erb]: This line starts with an h4 tag and ends with an h3 tag.

File: config/routes.rb [hunk 2]
```diff
@@ -21,6 +22,9 @@
 
 get 'pipedrive/get_deals' => 'pipedrive#get_deals'
 
+get 'referral' => 'referral#index'
```
Comment (tmclark737) [inline config/routes.rb]: We can write this as resources :referrals

File: config/routes.rb [hunk 3]
```diff
@@ -21,6 +22,9 @@
...
+post 'referral' => 'referral#index'
```
Comment (tmclark737) [inline config/routes.rb]: post referral should go to the create page.

File: app/controllers/referral_controller.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+class ReferralController < ApplicationController
+  
+
+
+  def contact_us
+
+
+			@submitted = params["commit"]
```
Comment (tmclark737) [inline app/controllers/referral_controller.rb]: contact us is not a restful route.

Comment (tmclark737) [inline app/controllers/referral_controller.rb]: Don't need to assign instance variables. can just get from params.

File: app/controllers/referral_controller.rb [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+			@phone = params["phone"]
+			@special_request = params["special_request"]
+			@interest_consultation = params["interest_consultation"]
+			@interest_information = params["interest_information"]
+			@interest_call = params["interest_call"]
+			@interest_looking = params["interest_looking"]
+
+			# Enqueue a job to be performed as soon the queueing system is
+			# free.
+			WebformEmailJob.set(wait: 41.minutes).perform_later(@name, @email, @phone, @special_request, @interest_consultation, @interest_information, @interest_call, @interest_looking)
```
Comment (tmclark737) [inline app/controllers/referral_controller.rb]: it is uncommon to make separate variables, when you can just use a hash.

Comment (markdfrey) [inline app/controllers/referral_controller.rb]: Thanks Ty. I'll take a closer look at all of these later today. 

Do I change things in my code or are these recommended edits that I accept on Git?

There are surely some other edits to make esthetically / content wise, including referral candy code if needed, and maybe others based on having mostly copied the contact us page. 

-Mark

**Pull request closed without merging**